user/models.py

```diff
@@ -0,0 +1,98 @@
+from django.db import models
+from django.db.models.fields.related import ForeignKey
+
+# 유저
+class User(models.Model):
+    user_id         = models.CharField(max_length=20)
+    user_password   = models.CharField(max_length=300)
+    user_name       = models.CharField(max_length=20)
+    email           = models.EmailField(max_length=50)
+    cell_phone      = models.CharField(max_length=20)
+    home_phone      = models.CharField(max_length=20, null=True)
+    user_address    = models.CharField(max_length=300, null=True)
```

Comment:
```suggestion
    account         = models.CharField(max_length=20)
    password        = models.CharField(max_length=300)
    name            = models.CharField(max_length=20)
    email           = models.EmailField(max_length=50)
    cell_phone      = models.CharField(max_length=20)
    home_phone      = models.CharField(max_length=20, null=True)
    address         = models.CharField(max_length=300, null=True)
```
가급적 컬럼에는 테이블 명과 중복되는 명칭을 지양하도록 해주세요.

Reply:
넵 수정했습니다!